content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

```diff
@@ -41,7 +41,7 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
 {% endif %}
 
 {% data reusables.enterprise_site_admin_settings.sign-in %}
-1. In the administrative shell, enable the {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data variables.product.prodname_dependabot_short %}{% else %}security{% endif %} alerts for vulnerable dependencies on {% data variables.product.product_location_enterprise %}:
+1. In the [administrative SSH shell](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh#enabling-access-to-the-administrative-shell-via-ssh), enable the {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data variables.product.prodname_dependabot_short %}{% else %}security{% endif %} alerts for vulnerable dependencies on {% data variables.product.product_location_enterprise %}:
```

Comment:
I realize this is an in-line link which is not preferred, but it felt a bit long as a "For more information on SSH access:" type call out.

Reply:
👋 @djdefi I agree that it'd be too long to use add a `For more info` into the procedural step, but we avoid inline links to help with localization, accessibility, and ensure users always understand what they're going to click on. What if, instead of using an inline link, you added a note after the shell command on line 47. Something like:

```
{{#note}}

**Note**: For more information about enabling access to the administrative shell via SSH, see "[Enabling alerts for vulnerable dependencies on GitHub Enterprise Server](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh#enabling-access-to-the-administrative-shell-via-ssh)."

{{/note}}
```